dask/tests/test_order.py

```diff
@@ -751,7 +751,7 @@ def test_order_with_equal_dependents(abcde):
             val = o[(x, 6, i, 1)] - o[(x, 6, i, 0)]
             assert val > 0  # ideally, val == 2
             total += val
-    assert total <= 32  # ideally, this should be 2 * 16 = 32
+    assert total <= 110  # ideally, this should be 2 * 16 = 32
```

Comment:
This bound is worse than before, but the other bounds below are better.  I don't think this change is critical.  Here is the new ordering for this graph:

![5859](https://user-images.githubusercontent.com/2058401/127804470-c63528d8-4945-4565-a2fd-ed7a8e5e4307.png)